lib/console.js

```diff
@@ -35,20 +37,34 @@ function Console(stdout, stderr) {
   }
 }
 
+function write(ignoreErrors, stream, string) {
+  if (!ignoreErrors) return stream.write(string);
+
+  try {
+    stream.write(string, (err) => {
+      if (err) {
+        stream.once('error', () => {});
+      }
+    });
```

Comment:
> The error event is emitted after the callback, so I don’t think that works.

Then this PR creates a memory leak, and after a single `console.log` is used, any `'error'` event will be swallowed.